src/main/java/ua/danit/final_project/entities/User.java

```diff
@@ -16,6 +16,7 @@
 import javax.persistence.JoinColumn;
 import javax.persistence.JoinTable;
 import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'javax.persistence.OneToMany'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=85&issues=AWWG_p72X2sY87p04Y76&open=AWWG_p72X2sY87p04Y76)

[ISSUE_KEY]: AWWG_p72X2sY87p04Y76